src/omv/sensor.c

```diff
@@ -433,7 +433,11 @@ int sensor_reset()
     sensor.hmirror       = false;
     sensor.vflip         = false;
     sensor.transpose     = false;
+    #if MICROPY_PY_IMU
+    sensor.auto_rotation = sensor.chip_id == OV7690_ID;
```

Comment:
Maybe it's better to move this to ov7690_init ?

Reply:
It can't be in init because reset would clear it...

Reply:
Oh, I see... like the color_palette.

Reply:
No, I think it makes sense in reset...

Reply:
I meant here:
https://github.com/openmv/openmv/blob/master/src/omv/ov7690.c#L616

Reply:
It doesn't really make sense to set it there. Instead can you merge the PR and then create a micro target and then we it being set correctly into the micro target.